Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>I'm not sure what the datestep means. Is it a meaningful value? If not, maybe we could just give the rows an index?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>